### ISSUE TITLE
[Darrowshire] Problems with envelopes from the quest "Field Duty"

### ISSUE THREAD

**Issue by nukul4r** [LightsHope/issues#47]
_The server or game patch that your issue pertains to:_
Darrowshire 1.10

_A clear, concise description of the current behaviour._
This concerns the envelopes gotten from the quest "Field Duty" (https://classicdb.ch/?quest=8731) in Silithus. I selected the "Tactical Assignment" envelope, and opened it. It only showed an empty loot window, which I could close. When I did that, the envelope didn't disappear, and showed an empty loot window again. I destroyed the envelope and repeated the quest a few more times, each time I only got the empty loot window. From a previous quest I also had a "Followup Tactical Assignment" (https://classicdb.ch/?item=21133) which showed the same behaviour.

_A description of the expected behaviour._
The envelopes should contain a scroll of the type "Tactical Task Briefing" (https://classicdb.ch/?search=tactical+task+briefing) which starts a quest. Also the envelope should disappear once looted. This worked before (that's how I got the followup assignment), but stopped working, I noticed it when trying to do the quests on Monday, Feb. 19th.

**Comment by nukul4r** [comment]
Regarding being on the quest, that is a possibility. I had the case before where I had the task briefing scroll in my inventory, and it showed me a new one, but wouldn't let me loot it. Now that you mention it, I will try to get some more of them, and cancel all tactical quests before opening. This somehow seems the same behaviour with artifacts in AQ40, where you can't loot them (i.e. they don't even drop for you) if you accepted the quest.

Thanks a lot, I will try this out as soon as I can get some more.

**Comment by Cabro** [comment]
Yes, it's the same bug where you don't see some quest starting items if you already completed the quests them. #privatelife
